crates/iota-types/src/unit_tests/utils.rs

```diff
@@ -120,6 +120,22 @@ pub fn to_sender_signed_transaction(
     to_sender_signed_transaction_with_multi_signers(data, vec![signer])
 }
 
+pub fn to_sender_signed_transaction_with_aa_sponsor(
+    data: TransactionData,
+    authenticator: GenericSignature,
+    sponsor: Option<&dyn Signer<Signature>>,
```

Comment:
After my previous suggestion I realized this can be made more generic and maybe reused in other context. So I suggest a renaming:
```suggestion
pub fn to_sender_signed_transaction_with_optional_sponsor(
    data: TransactionData,
    sender_signature: GenericSignature,
    sponsor_signer_opt: Option<&dyn Signer<Signature>>,
```

Reply:
Fixed 53707743f48ced153bd557331836d6c007907b1b